sql_generators/active_users_aggregates/__init__.py

```diff
@@ -0,0 +1,116 @@
+"""Generate active users aggregates per app."""
+import os
+from enum import Enum
+from pathlib import Path
+
+import click
+from jinja2 import Environment, FileSystemLoader
+
+from bigquery_etl.cli.utils import use_cloud_function_option
+from bigquery_etl.format_sql.formatter import reformat
+from bigquery_etl.util.common import render, write_sql
+
+THIS_PATH = Path(os.path.dirname(__file__))
+
+
+class Browsers(Enum):
+    """Enumeration with browser names and equivalent dataset names."""
+
+    firefox_desktop = "Firefox Desktop"
+    fenix = "Fenix"
+    focus_ios = "Focus iOS"
+    focus_android = "Focus Android"
+    firefox_ios = "Firefox iOS"
+    klar_ios = "Klar iOS"
+
+
+@click.command()
+@click.option(
+    "--output-dir",
+    "--output_dir",
+    help="Output directory generated SQL is written to",
+    type=click.Path(file_okay=False),
+    default=Path("sql"),
+)
+@click.option(
+    "--target-project",
+    "--target_project",
+    help="Google Cloud project ID",
+    default="moz-fx-data-shared-prod",
+)
+@use_cloud_function_option
+def generate(target_project, output_dir, use_cloud_function):
+    """Generate per-app queries, views and metadata for active users and search counts aggregates.
+
+    The parent folders will be created if not existing and existing files will be overwritten.
+    """
+    env = Environment(loader=FileSystemLoader(str(THIS_PATH / "templates")))
+    metadata_template = env.get_template("metadata.yaml")
+    mobile_query_template = env.get_template("mobile_query.sql")
+    desktop_query_template = env.get_template("desktop_query.sql")
+    view_template = env.get_template("view.sql")
+
+    for browser in Browsers:
+        metadata_output_dir = Path(
+            f"{output_dir}/{target_project}/{browser.name}_derived/"
```

Comment:
How to test before merging the PR, that files are generated in the correct path?